clustered/client/src/main/java/org/ehcache/clustered/client/internal/store/ClusteredStore.java

```diff
@@ -677,15 +779,24 @@ public ClusteredCacheIdentifier getCacheIdentifier() {
     public Consistency getConsistency() {
       return consistency;
     }
+
+    public int getConcurrency() {
+      return concurrency;
+    }
   }
 
+  //added @SuppressFBWarnings to get rid of findBugs error which recommended making instance variables static
+  @SuppressFBWarnings
   private static final class ClusteredStoreStatsSettings {
```

Comment:
if I remove the ClusteredStoreStatsSettings class what is the best way to call associate()?
line 119:
StatisticsManager.associate(clusteredStoreStatsSettings).withParent(this);